src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -17,151 +25,171 @@
         );
     }
 
     @Bean
     public PreSubmitCallbackHandler<AsylumCase> respondentEvidenceNotificationHandler(
-        @Qualifier("respondentEvidenceNotificationGenerator") NotificationGenerator notificationGenerator) {
+        @Qualifier("respondentEvidenceNotificationGenerator") List<NotificationGenerator> notificationGenerators) {
 
         return new NotificationHandler(
             (callbackStage, callback) ->
                 callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                && callback.getEvent() == Event.REQUEST_RESPONDENT_EVIDENCE,
-            notificationGenerator
+                    && callback.getEvent() == Event.REQUEST_RESPONDENT_EVIDENCE,
+            notificationGenerators
+
         );
     }
 
     @Bean
     public PreSubmitCallbackHandler<AsylumCase> respondentReviewNotificationHandler(
-        @Qualifier("respondentReviewNotificationGenerator") NotificationGenerator notificationGenerator) {
+        @Qualifier("respondentReviewNotificationGenerator") List<NotificationGenerator> notificationGenerators) {
 
         return new NotificationHandler(
             (callbackStage, callback) ->
                 callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                && callback.getEvent() == Event.REQUEST_RESPONDENT_REVIEW,
-            notificationGenerator
+                    && callback.getEvent() == Event.REQUEST_RESPONDENT_REVIEW,
+            notificationGenerators
         );
     }
 
     @Bean
-    public PreSubmitCallbackHandler<AsylumCase> submitAppealNotificationHandler(
-        @Qualifier("submitAppealNotificationGenerator") NotificationGenerator notificationGenerator) {
+    public PreSubmitCallbackHandler<AsylumCase> submitAppealAipNotificationHandler(
+        @Qualifier("submitAppealAipNotificationGenerator") List<NotificationGenerator> notificationGenerators
+    ) {
 
         return new NotificationHandler(
             (callbackStage, callback) ->
                 callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                && callback.getEvent() == Event.SUBMIT_APPEAL,
-            notificationGenerator
+                    && callback.getEvent() == Event.SUBMIT_APPEAL
+                    && callback.getCaseDetails().getCaseData()
+                    .read(JOURNEY_TYPE, JourneyType.class).isPresent()
+                    && (callback.getCaseDetails().getCaseData()
+                    .read(JOURNEY_TYPE, JourneyType.class).get() == AIP),
+            notificationGenerators
+        );
+    }
+
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> submitAppealRepNotificationHandler(
+        @Qualifier("submitAppealRepNotificationGenerator") List<NotificationGenerator> notificationGenerators
+    ) {
+
+        return new NotificationHandler(
+            (callbackStage, callback) ->
+                callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+                    && callback.getEvent() == Event.SUBMIT_APPEAL,
```

Comment:
one small comment, you need to prevent this handler for being invoked for AIP use case by adding condition:
```
callback.getCaseDetails().getCaseData()
                    .read(JOURNEY_TYPE, JourneyType.class)
                    .map(type -> type == REP).orElse(false),
```
otherwise you will get two email notifications.
PS. 
There is duplication watchdog deeper in code so you do not see this in functional tests results

Reply:
Thank you for the suggestion applied change on next commit